docs/site/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "marked": "^0.3.6",
+    "material-ui-icons": "^1.0.0-alpha.2",
```

Comment:
@oliviertassinari - the icons aren't in the repository I guess we could add them, but unless there is major change, there is no maintenance cost.